### ISSUE TITLE
Firefox Quantum

### ISSUE THREAD

**Issue by Anbima** [roytam1/mozilla45esr#11]
Feodor2 has ported Firefox Quantum for Windows XP. Please see the following page:
https://github.com/Feodor2/Mypal

Unfortunately, Feodor2 will not be able to continue the project because he is from Ukraine.

But if Feodor2 has managed to port Firefox Quantum for Windows XP, then this should also be feasible for other developers. 
Hence the question to you, roatam1: Would you tackle the project so that we get a current browser for Windows XP?

**Comment by roytam1** [comment]
unless he release his effort or no one can continue.

**Comment by Anbima** [comment]
I'm afraid that we will not get the source code.
Can not you start a new, own project with Firefox Quantum?

Feodor2 wrote that he did it with the rust compiler. The source code of Firefox Quantum is publicly available.

He wanted to do it with Quantum 68, 78 and 91.

**Comment by roytam1** [comment]
> I'm afraid that we will not get the source code. Can not you start a new, own project with Firefox Quantum?
> 
> Feodor2 wrote that he did it with the rust compiler. The source code of Firefox Quantum is publicly available.
> 
> He wanted to do it with Quantum 68, 78 and 91.

changes are huge because mozilla removed lots of things

**Comment by Anbima** [comment]
That's right, but that's very good as the quantum is much better and faster. The first tests of Feodor2 were promising. You have to create it completely new from the original source code. But you would not have to go on the other browsers anymore, as the quantum would hit everyone.

I and many other XP users would be really grateful.

**Comment by Anbima** [comment]
Maybe you will continue to help you:
https://firefox-source-docs.mozilla.org/
